### DISCUSSION TITLE
Can't connect to OpenSSH node with session recording in proxy mode

### DISCUSSION THREAD

**Discussion by fcharlaix-opendsi** [gravitational/teleport#10014] [Q&A]
Hi,

I try the teleport solution and I need to make old node on Debian 9/8 joining the cluster, I setup the node and proxy for OpenSSH with [this documentation](https://goteleport.com/docs/server-access/guides/openssh/#set-up-openssh-recording-proxy-mode).

When I set `auth_service.session_recording` to `off` I can connect without problem, but with `proxy` like in the documentation I got this error:
```
$ tsh ssh -A --port=22 root@dev-something.infra.lan
ERROR: failed connecting to node dev-something.infra.lan. agent forwarding required in proxy recording mode
```

This seems related to #9213 and #9219, so I build the `v8.1.2-dev.1` tag and use it as a server and client, but I have the same issue.

Did I need to open an issue, or I make a mistake on the configuration ?
Thanks !

**Comment by klizhentas** [comment]
@fcharlaix-opendsi you need to make sure that agent forwarding is allowed in RBAC for your user (tsh status should show it) and you enabled agent forwarding:

https://goteleport.com/docs/server-access/guides/openssh/#setup-ssh-agent-forwarding

**Comment by webvictim** [comment]
You should also check that your local `ssh-agent` is running and shows a Teleport key and certificate after you log in with `tsh login`:

```
$ ssh-add -l
2048 SHA256:buY1gyLRRRBE2H/tVjqyCopZoD87kO2rQQRlsqxJiuU teleport:webvictim (RSA-CERT)
2048 SHA256:buY1gyLRRRBE2H/tVjqyCopZoD87kO2rQQRlsqxJiuU teleport:webvictim (RSA)
```

If it's still not working, please add `--debug` to your `tsh` command and share the output.

**Comment by c4milo** [comment]
I'm running into the same in Teleport v11.2.1 git: go1.19.5
